### DISCUSSION TITLE
Disconnecting SimBrief connection and deleting settings: how?

### DISCUSSION THREAD

**Discussion by tup61** [flybywiresim/aircraft#6513] [Help]
For my own personal testing purposes I uninstalled the FBW as instructed on the FBW site. I even went as far as to delete all other (undocumented) FBW folders and files and each and every entry in the register. After reinstalling the FBW A32NX brand new I was surprised to see my settings were still the same and, even more surprising, the SimBrief account was still connected...!

I would like to know how I can uninstall the FBW A32NX completely so ALL settings are default again and the SimBrief connection is gone!

Apparently all these settings are saved somewhere but I have no clue where. As I said I deleted absolutely EVERYTHING!

**Comment by tup61** [comment]
BUMP 

Anybody out there?

**Reply by tracernz** [reply]
Not sure where they're saved. MSFS saves the data store on disk, not us (unfortunately the MSFS SDK doesn't allow us to write files to disk from JS).

**Reply by tup61** [reply]
Thanks for the reply! So okay, I have to dig into the MSFS files to see where these settings are saved! Strange that FBW doesn't have control over this! Also strange no one asked this before apparently...? ;) But anyway, I will change a setting and search for files by date somehow. I'm curious if it is even possible to find the files in which the settings are saved...

**Reply by tracernz** [reply]
> Thanks for the reply! So okay, I have to dig into the MSFS files to see where these settings are saved! Strange that FBW doesn't have control over this! Also strange no one asked this before apparently...? ;) But anyway, I will change a setting and search for files by date somehow. I'm curious if it is even possible to find the files in which the settings are saved...

It would be a good idea for us to have a way to reset everything… I’ll keep that in mind.

**Comment by yuriks59** [comment]
I'm a little off topic, but I would like to have the simbrief plan loaded in the tablet in the stable version. Otherwise, the button is active, but there is no plan download, as in the developer version.
